package.json

```diff
@@ -17,12 +17,12 @@
   "grunt-shell": "^2.1.0",
   "grunt-wp-deploy": "^1.3.0",
   "husky": "^0.14.3",
-  "workbox-cli": "4.0.0-beta.2",
+  "workbox-cli": "4.0.0-rc.2",
   "wp-dev-lib": "github:xwp/wp-dev-lib#1.0.0"
  },
  "scripts": {
   "install": "composer install",
-  "precommit": "./node_modules/wp-dev-lib/scripts/pre-commit",
+  "precommit": "if command -v lando >/dev/null 2>&1; then lando ssh -c 'bash ./node_modules/wp-dev-lib/scripts/pre-commit'; else ./node_modules/wp-dev-lib/scripts/pre-commit; fi",
```

Comment:
@felixarntz what do you think of this?

Reply:
I guess that would work, but tbh I'm really not a fan of putting environment-specific details into a general project repository, unless that environment has something to do with standardization (e.g. `editorconfig`).

Reply:
Struggling to think of an alternative way to keep Git running on the host machine but to run all Git hooks in the container.